### DISCUSSION TITLE
Chromium libraries - Apache or GPLv3 components interact with or operate in the same memory space as the GPL v2-licensed components?

### DISCUSSION THREAD

**Discussion by dealsweb** [cefsharp/CefSharp#4749] [General]
I have a question about utilizing Chromium libraries. The licenses for different libraries incorporated within Chromium include a combination of Apache, GPL2, and GPL3. Apache components are intertwined with GPLv2, and GPLv3 is listed alongside GPLv2. Has there been a validation or confirmation ensuring that none of these Apache or GPLv3 components interact with or operate in the same memory space as the GPL v2-licensed components?

**Comment by amaitland** [comment]
This is really a question for the `Chromium` development team. You'll need to ask them for guidance.
